pkg/manifests/manifests.go

```diff
@@ -147,6 +148,22 @@ func APICRD() (*apiextensionv1.CustomResourceDefinition, error) {
 	return crd, nil
 }
 
+// APINFDCRD will return a newer CRD version that being used by NFD
+// Eventually we should use only the newer version
+// but we need to wait until scheduler plugin and RTE will catch up
+func APINFDCRD() (*apiextensionv1.CustomResourceDefinition, error) {
+	obj, err := loadObject("yaml/api/crd_for_nfd.yaml")
```

Comment:
I'm against this. We should bump the APIs, rebuild and only then consume from deployer.